esphome/onju-voice-microwakeword.yaml

```diff
@@ -235,18 +239,16 @@ voice_assistant:
         green: 100%
         effect: speaking
   on_end:
-    - delay: 200ms
     - wait_until:
         not:
-          media_player.is_playing: onju_out
+          media_player.is_playing:
```

Comment:
I just noticed that I didn't specify `onju_out` here explicitly. On the other hand, in other locations that is also not specified explicity, thus it is fine?

Reply:
It works if it's the only `media_player`